pkg/stack/options.go

```diff
@@ -60,44 +63,62 @@ func FromOptions() (*Stack, error) {
 	return FromFile(configPath)
 }
 
-func FunctionFromHandler(h, stackDir string) Function {
-	rt, _ := runtime.NewRunTimeFromHandler(h)
-	fn := Function{
-		ComputeUnit: ComputeUnit{Name: rt.ContainerName()},
-		Handler:     h,
+func EnsureRuntimeDefaults() bool {
+	defaults := map[string]map[string]interface{}{
+		"ts": {
+			"functionglob": "functions/*.ts",
+		},
+		"go": {
+			"functionglob": "functions/*/*.go",
+		},
```

Comment:
this is really good!

I think we'll need to consider allowing `.nitric-config` at the project level as well, as I could see this kind of configuration being specific to a project as well as target configuration within a project.